encryption.rb

```diff
@@ -1,4 +1,28 @@
 #Encryption and decryption of messages
   class Encryption
 
+    def caesar_cipher(string, shift)
+      alphabet  = Array('a'..'z')
+      non_caps  = Hash[alphabet.zip(alphabet.rotate(shift))]
+
+      alphabet = Array('A'..'Z')
+      caps     = Hash[alphabet.zip(alphabet.rotate(shift))]
+
+      encrypter = non_caps.merge(caps)
+
+      string.chars.map { |c| encrypter.fetch(c, c) }
+    end
+
+    def caesar_cipher2(string, shift)
+      alphabet  = Array('a'..'z')
+      non_caps  = Hash[alphabet.zip(alphabet.rotate(shift))]
+
+      alphabet = Array('A'..'Z')
+      caps     = Hash[alphabet.zip(alphabet.rotate(shift))]
+
+      encrypter = non_caps.merge(caps)
+
+      string.chars.map { |c| encrypter.fetch(c, c) }
+    end
+
```

Comment:
Rename methods, make sure that puts is implemented here instead of doing that in main class.